ansible/testbed-cli.sh

```diff
@@ -332,8 +332,7 @@ function add_topo
     fi
 
     if [[ $topo == *"t2"* ]]; then
-      ansible-playbook -i ${inv_name} testbed_config_vchassis.yml --vault-password-file="$passfile" -l "$duts" -e testbed_name="$testbed_name" \
-      -e topo="$topo" -e testbed_file=$tbfile -e vm_file=$vmfile  -e server="$server" $@
+      ansible-playbook -i ${inv_name} testbed_config_vchassis.yml --vault-password-file="$passfile" -l "$duts" -e topo="$topo"
```

Comment:
I think the password file is saved in the variable `${passwd}` in `add_topo`, not sure how important this is for this call.